lib/events-scraper/event.rb

```diff
@@ -65,7 +65,7 @@ def remote_document
       uri = "#{WEBSITE}#{@link}?do=export_xhtmlbody"
       print "\n #{@index+1} ".yellow
       print "#{WEBSITE}#{@link}.html ".blue
-      Nokogiri::HTML(open(uri))
+      Nokogiri::HTML(open(uri), nil,"UTF-8")
```

Comment:
in that case, would it be enough to just set UTF-8 when writing the output file?